src/com/theironyard/novauc/City.java

```diff
@@ -0,0 +1,12 @@
+package com.theironyard.novauc;
+
+/**
+ * Created by Merlin on 2/7/17.
+ */
+public class City {
+    String NameOfCity;
+    String MayorName;
+    int population;
+    int yearEstablished;
+    boolean PopulationIncreasedLastYear;
+}
```

Comment:
variable names follow the format of starting the first word with a lowercase letter and the second word starting with an uppercase letter.  So line 10 is correct.  Line 11 should be populationIncreasedLastYear.  Follow that format in the future.  Overall good job